internal/commands/install.go

```diff
@@ -32,22 +32,25 @@ const (
 	nameKindReference
 )
 
-const longDescription = `Install the application on either Swarm or Kubernetes.
-Bundle name is optional, and can:
-- be empty and resolve to any *.dockerapp in working directory
-- be a BUNDLE file path and resolve to any *.dockerapp file or dir, or any CNAB file (signed or unsigned)
-- match a bundle name in the local duffle bundle repository
-- refer to a CNAB in a container registry
-`
+const longDescription = `Install an application.
+By default, the application definition in the current directory will be
+installed. The APP_NAME can also be:
+- a path to a Docker Application definition (.dockerapp) or a CNAB bundle.json
+- a registry Application Package reference`
+
+const example = `$ docker app install myapp.dockerapp --name myapp --target-context=mycluster
+$ docker app install myrepo/myapp:mytag --name myapp --target-context=mycluster
+$ docker app install bundle.json --name myapp --credential-set=mycredentials.yml`
 
 func installCmd(dockerCli command.Cli) *cobra.Command {
 	var opts installOptions
 
 	cmd := &cobra.Command{
-		Use:     "install [<bundle name>] [OPTIONS]",
+		Use:     "install [APP_NAME] [--name INSTALLATION_NAME] [--target-context TARGET_CONTEXT] [OPTIONS]",
```

Comment:
APP_NAME shouldn't have brackets.

Reply:
It's optional because it picks up the `.dockerapp` in the current directory if nothing is specified.